directjngine/src/main/java/com/softwarementors/extjs/djn/jscodegen/CodeFileGenerator.java

```diff
@@ -166,9 +166,13 @@ private static boolean fileContentEquals(File file, String code) throws IOExcept
     }
     return false;
   }
-  
+
   private static String getDebugFileName( String file ) {
-    String result = file.replace( ".js", "-debug.js");
-    return result;
+    int extensionIndex = file.lastIndexOf( ".js" );
```

Comment:
`file.replaceFirst(".js$", "-debug.js");` might be a better way to express this.

Reply:
Much better! 👍 
Applied in 9eb5223